openedx/core/djangoapps/course_groups/management/commands/convert_cohort_format.py

```diff
@@ -20,10 +20,15 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         store = modulestore()
         for course in store.get_courses():
-            if course.cohort_config and 'inline_discussions_cohorting_default' in course.cohort_config:
+            if course.cohort_config:
                 print "Updating affected course: %s" % unicode(course.location)
-                course.cohort_config['always_cohort_inline_discussions'] = course.cohort_config[
-                    'inline_discussions_cohorting_default'
-                ]
+                if 'inline_discussions_cohorting_default' in course.cohort_config:
+                    course.cohort_config['always_cohort_inline_discussions'] = course.cohort_config[
+                        'inline_discussions_cohorting_default'
+                    ]
+                inlines = course.cohort_config.get('cohorted_inline_discussions', [])
+                course_discussions = course.cohort_config.get('cohorted_course_wide_discussions', [])
+                if inlines or course_discussions:
+                    course.cohort_config['cohorted_discussions'] = inlines + course_discussions
```

Comment:
It looks like this will replace the `cohorted_discussions` value, even if it contained unique discussion IDs that weren't in the deprecated fields. But I don't think that should ever occur, since updates with the new version get made in the DB and not the course object, so this should be fine.